extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -352,13 +350,24 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.buildCarouselSlides_(list);
       return this.mutateElement(() => {
         this.carouselContainer_.appendChild(this.carousel_);
-        const hideControls = list.length == 1;
-        this.controlsContainer_.classList.toggle('i-amphtml-ghost',
-            hideControls);
+        this.maybeEnableMultipleItemControls_(list.length);
       });
     });
   }
 
+  /**
+   * @param {number} itemLength
+   * @private
+   */
+  maybeEnableMultipleItemControls_(itemLength) {
+    const isDisabled = itemLength <= 1;
+    const {controlsContainer_} = this;
+    const ghost = 'i-amphtml-ghost';
+    controlsContainer_.classList.toggle(ghost, isDisabled);
```

Comment:
Can we double check that this doesn't accidentally disable the X button on Firefox? Maybe instead of using ghost on `controlsContainer_` we can just use it on left, right and gallery button specifically. There should be a container div for the left and right arrows as well.